### PR TITLE
gcalcli: init at 3.4.0

### PR DESCRIPTION
**Pull request by nocoolnametom**
###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/gcalcli/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgs, lib, python, pythonPackages }:
```
Comment (FRidh) [inline pkgs/applications/misc/gcalcli/default.nix]: - `stdenv` is not used.
- never pass in `pkgs`.
- pass in either `python` or `pythonPackages`, no need for both. You can access `pythonPackages` via `python.pkgs` and the interpreter with `pythonPackages.python`.

File: pkgs/applications/misc/gcalcli/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    oauth2client
+    parsedatetime
+    six
+    vobject
+  ] ++ lib.optional (!pythonPackages.isPy3k) futures;
+
+  meta = with lib; {
+    homepage = https://github.com/insanum/gcalcli;
+    description = "CLI for Google Calendar";
+    license = licenses.mit;
```
Comment (FRidh) [inline pkgs/applications/misc/gcalcli/default.nix]: are you going to maintain this package?

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/gcalcli/default.nix (hunk 1 above)
Comment (nocoolnametom) [inline pkgs/applications/misc/gcalcli/default.nix]: Thanks for the feedback. I'm still new to Nix... I'll get that updated.

File: pkgs/applications/misc/gcalcli/default.nix (hunk 2 above)
Comment (nocoolnametom) [inline pkgs/applications/misc/gcalcli/default.nix]: I can.

**Review by nocoolnametom — COMMENTED** [review]
If this all looks good then I think this is ready to go.

**Pull request merged**